website/docs/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -44,20 +44,16 @@ To view and manage a workspace's variables, go to the workspace and click the **
 
 The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace. This is where you can add, edit, and delete workspace-specific variables. You can also apply and remove variable sets from the workspace.
 
-![Screenshot: The variables page for a workspace](/img/docs/vars.png)
-
 ### Add a Variable
 
 To add a variable:
 
 1. Go to the workspace **Variables** page and click **+ Add variable** in the **Workspace Variables** section.
 
-1. Choose a variable category (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable key, value, and optional description. Check the **HCL** checkbox to enter a value in HashiCorp Configuration Language.
+1. Choose a variable category (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable key, value, and optional description. If the variable is a Terraform variable, you can check the **HCL** checkbox to enter a value in HashiCorp Configuration Language. You can only use HCL for Terraform variables, but not for environment variables.
```

Comment:
What do we think about this? I was trying to convey that same clarification (which I think is a good one about how this only applies to Terraform variables) but make it shorter.

```suggestion
1. Choose a variable category (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable key, value, and optional description. For Terraform variables only, you can check the **HCL** checkbox to enter a value in HashiCorp Configuration Language. 
```